kratos/processes/integration_values_extrapolation_to_nodes_process.cpp

```diff
@@ -217,6 +217,47 @@ void IntegrationValuesExtrapolationToNodesProcess::ExecuteFinalizeSolutionStep()
             }
         }
     }
+
+    // Assemble nodal data
+    if (mExtrapolateNonHistorical)
+    {
+        for (const auto p_var : mDoubleVariable)
+        {
+            mrModelPart.GetCommunicator().AssembleCurrentData(*p_var);
+        }
+        for (const auto p_var : mArrayVariable)
+        {
+            mrModelPart.GetCommunicator().AssembleCurrentData(*p_var);
+        }
+        for (const auto p_var : mVectorVariable)
```

Comment:
```suggestion
        for (const auto p_var : mVectorVariable) {
```

should be on same line